src/server/handlers.rs

```diff
@@ -762,3 +747,158 @@ impl<P: Publisher> Handler<P> {
         self.send_reply(reply, response).await;
     }
 }
+
+pub async fn open_schema_file() -> anyhow::Result<BufReader<File>> {
+    let schema_file = File::open("./oam/oam.schema.json")?;
```

Comment:
I don't see this file committed, did it miss the git commit?

Reply:
No I forgot to commit it. Sorry.